scripts/squidex-minimal-azure-arm.json

```diff
@@ -0,0 +1,277 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "containerName": {
+      "defaultValue": "squidex",
+      "type": "String",
+      "metadata": {
+        "description": "The name of the Azure Container Instance Group"
```

Comment:
Can you change this to "Name of your deployment and prefix for all container names."?

Reply:
Done